### DISCUSSION TITLE
Initial concentration in electrodes

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#3774] [Q&A]
I am parameterizing a cell; I want to know how to calculate the initial concentration in the electrodes. I know the maximum concentrations in electrodes and the x_min and x_max of each electrode.
Thank you.

**Comment by valentinsulzer** [comment]
For negative electrode, `c_n_init = c_n_max * x_max`. For positive, `c_p_init = c_p_max * y_min`. (Those are for starting at 100% SOC)

**Reply by kumaryash7** [reply]
What if I want to start to my cell cycling at 50 % SOC? 
and also how is it different from defining (initial_soc=1) inside sim.solve?
Thank you
